daemon/Makefile.am

```diff
@@ -4,6 +4,7 @@ gluster_blockd_SOURCES = gluster-blockd.c
 
 gluster_blockd_CFLAGS = $(GFAPI_CFLAGS) $(TIRPC_CFLAGS)                        \
                         -DDATADIR=\"$(localstatedir)\"                         \
+                        -DENABLE_SYSTEMD=\"$(USE_SYSTEMD)\"                    \
```

Comment:
This will work, but would it not be nicer to have #defines for features in `config.h`?

Reply:
@nixpanic 

Yeah, true. And the ENABLE_SYSTEMD maybe used in other c files in future and will fix it.

Thanks.